Categories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h2>Coming Soon</h2>
+    <button type="button">Check Status</button>
+  </div>
+);
+
+export default Categories;
```

Comment:
- [ ] Defines CHECK_STATUS as action type : The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.
- [ ] Creates a reducer and exports it as default.

* in case `action.type` is `CHECK_STATUS` the function returns a string "Under construction".
* if the `action.type` is different, it returns the state with no changes. 
- Kindly implement it